embassy-executor-macros/src/macros/main.rs

```diff
@@ -123,6 +139,32 @@ pub fn run(args: TokenStream, item: TokenStream, arch: &Arch) -> TokenStream {
         },
     };
 
+    let executor = match (
+        args.executor.as_deref(),
+        arch.default_entry.as_deref(),
+        arch.executor_required,
+    ) {
+        (None, None, true) => {
```

Comment:
this means on arch unknown if the user doesn't specify the executor but does specify the entry it'll silently use the default instead of erroring. is this correct?

it seems to me it should error.

Reply:
Missed that one, thanks